### PR TITLE
default

### PR DESCRIPTION
**Pull request by karpp**

### CODE REVIEW

File: default/app.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+runtime: python37 # or python37 for Python 3.7
```
Comment (niedziol) [inline default/app.yaml:1]: Change to python38.

File: default/app.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+instance_class: F1
```
Comment (mgmacias95) [inline default/app.yaml:3]: Not necessary since this is the default value

File: default/app.yaml (hunk 1 above)
Comment (karpp) [reply default/app.yaml]: Resolved

File: default/app.yaml (hunk 2 above)
Comment (karpp) [reply default/app.yaml]: Fixed

**Pull request merged**